File: ObjC/PryNotificacionObjc02/PryNotificacionObjc02/FirstViewController.m
//
//  FirstViewController.m
//  PryNotificacionObjc02
//
//  Created by Henry AT on 4/30/15.
//  Copyright (c) 2015 Apps4s. All rights reserved.
//

#import "FirstViewController.h"

@interface FirstViewController ()

@end

@implementation FirstViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
  [super didReceiveMemoryWarning];
  // Dispose of any resources that can be recreated.
}


- (IBAction)btnContador01:(id)sender {
  [[NSNotificationCenter defaultCenter] postNotificationName:@"Test01" object:self];
  
}

- (IBAction)btnContador02:(id)sender {
  [[NSNotificationCenter defaultCenter] postNotificationName:@"Test02" object:self];
  
}






































@end

File: Swift/PryNotificacionSwift02/PryNotificacionSwift02/FirstViewController.swift
//
//  FirstViewController.swift
//  PryNotificacionSwift02
//
//  Created by Henry AT on 5/1/15.
//  Copyright (c) 2015 Apps4s. All rights reserved.
//

import UIKit

class FirstViewController: UIViewController {

  override func viewDidLoad() {
    super.viewDidLoad()
    // Do any additional setup after loading the view, typically from a nib.
  }

  override func didReceiveMemoryWarning() {
    super.didReceiveMemoryWarning()
    // Dispose of any resources that can be recreated.
  }

// = = = = = = = = = = = = = = = = = = = = = 
  
  @IBAction func btnContador01(sender: AnyObject) {
    NSNotificationCenter.defaultCenter().postNotificationName("Test01", object: nil)
  }
  
  
  @IBAction func btnContador02(sender: AnyObject) {
    NSNotificationCenter.defaultCenter().postNotificationName("Test02", object: nil)
    println("Hola")
  }
  
}

